external/js/kyber/README.md

```diff
@@ -0,0 +1,48 @@
+KyberJS
```

Comment:
You need two disclaimers in here in big bold writing: 1. This is developmental, and not ready for protecting production data. 2. This is not a constant time implementation, and likely has timing side channels that can be attacked.